### PR TITLE
Redesign/tab status title

### PR DESCRIPTION
**Pull request by pallabez**
### Dev Tested:

- [x] Yes
- [ ] No

### Put the Feature behind Feature Flag

- [ ] Yes
- [x] No

### Follow-up Issues (if any):
image

> NOTE: After your PR is merged in main please verify its working on prod and raise a dev to main PR

### CODE REVIEW

File: components/PopupAssignee.vue [hunk 1]
```diff
@@ -1,50 +1,91 @@
...
+  isMenuOpen: false
 })
+const filteredUsers = computed(() => fuzzySearch(users, state.search))
+
+function fuzzySearch(users: User[] , searchText: string) {
+  if (!searchText) {      // No filter on user if searchText is empty
+    return users;
+  }
+
+  const searchRegex = new RegExp(searchText.split('').join('.*?'), 'i');
```
Comment (bhtibrewal) [inline components/PopupAssignee.vue:80]: please add explanation for this

File: components/PopupAssignee.vue [hunk 2]
```diff
@@ -59,7 +100,18 @@ const users = userStore.all().map(user => {
...
+  height: 500px;
+  overflow: visible;
+}
+
+.autocomplete__list {
+  height: 300px;
+}
+
+.autocomplete__list-item:hover {
+  background-color: rgb(218, 218, 218);
```
Comment (bhtibrewal) [inline components/PopupAssignee.vue:114]: use color variables

File: components/PopupAssignee.vue (hunk 1 above)
Comment (pallabez) [reply components/PopupAssignee.vue]: Will do so in a different PR. This code is not part of this PR as the original PR has been merged.

File: components/PopupAssignee.vue (hunk 2 above)
Comment (pallabez) [reply components/PopupAssignee.vue]: Not part of this PR

File: components/PopupAssignee.vue (hunk 1 above)
Comment (pallabez) [reply components/PopupAssignee.vue]: Should now not show up in this PR

**Review by prakashchoudhary07 — APPROVED** [review]

**Pull request merged**